Add

AngryFace97 opened the pull request:
+ Szkielet html sekcji about
+ Images

AngryFace97 committed:
Add

MartynLazarewicz commented on src/partials/about.html:
```diff
@@ -0,0 +1,17 @@
+    </div>
+    <div class="about__cow-and-text">
+        <div class="about__cow-img">
+
```
nie podałes tagu img

MartynLazarewicz commented on src/partials/about.html:
```diff
@@ -0,0 +1,17 @@
+            <p class="about__description--first">Fusce ut velit laoreet, tempus arcu eu, molestie tortor. Nam vel justo cursus, faucibus lorem eget, egestas eros. Maecenas eleifend erat at justo fringilla.</p>
+            <p class="about__description--second">Curabitur lacinia enim at ex blandit, vel pellentesque odio elementum. Mauris rhoncus orci in imperdiet placerat. Vestibulum euismod nisl suscipit ligula volutpat, a feugiat urna maximus. Cras massa nibh, tincidunt.</p>
+            <p class="about__description--third">Aliquam erat volutpat. Aenean accumsan.</p>
+            <button type="button" class="about__btn">Read more &#62;</button>
```
strzałkę w prawo bedziemy uzywać jako svg uzywając <svg> <use> </use> <svg>

MartynLazarewicz commented on src/partials/about.html:
```diff
@@ -0,0 +1,17 @@
+<div class="about">
```
About powinna byc w tagu <section> a cały content w danej sekcji powinnien byc owinienty divem z klasą wrapper która bedzie wyznaczała szerokość maxymalną

AngryFace97 committed:
Commit

AngryFace97 committed:
Commit

AngryFace97 committed:
Add SCSS to section about

Mały problem z zdjęciem rozlanego mleka które jest w tle przy drugim paragrafie. Zdjęcie wygląda dobrze przy rozdzielczości 320px, ale przy powiększaniu ekranu, zdjecie pozostaje w miejscu.

SVG dodane na razie jako PNG

MartynLazarewicz committed:
Merge branch 'main' into feature/about
